xrootd/xrdproto/verifyw/verifyw.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package verifyw contains the structures describing verifyw request.
+// See xrootd protocol specification (http://xrootd.org/doc/dev45/XRdv310.pdf, p. 124) for details.
+package verifyw // import "go-hep.org/x/hep/xrootd/xrdproto/verifyw"
+
+import (
+	"encoding/binary"
+	"hash/crc32"
+
+	"go-hep.org/x/hep/xrootd/internal/xrdenc"
+	"go-hep.org/x/hep/xrootd/xrdfs"
+)
+
+// RequestID is the id of the request, it is sent as part of message.
+// See xrootd protocol specification for details: http://xrootd.org/doc/dev45/XRdv310.pdf, 2.3 Client Request Format.
+const RequestID uint16 = 3026
+
+// VerificationType identifies the checksum algorithm used.
+type VerificationType uint8
```

Comment:
as the name of the package is `verifyw`, I think we can drop the `Verification` part and have just:
```go
type Type uint8
```
much less stuttering.